File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        hashmap = {}
        for char in s:
            if char in hashmap:
                hashmap[char] = hashmap[char] + 1
            else:
                hashmap[char] = 1
        
        for char in t:
            if char in hashmap and hashmap[char] > 0:
                hashmap[char] = hashmap[char] - 1
            else:
                return False

        for value in hashmap.values():
            if value != 0:
                return False

        return True

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let hashMap = new Set(nums)
    let maxSquence = 0;
    for(let num of nums) {
        if(hashMap.has(nums - 1)){
            continue
        } else {
            let i = 1
            let currentMax = 1;
            while(true){
                if(hashMap.has(num + i)) {
                    currentMax += 1;
                    i++;
                } else {
                    break;
                }
            }
            if(maxSquence < currentMax) {
                maxSquence = currentMax;
            }
        }
    }
    return maxSquence
    }
}
